src/js/themes/base.js

```diff
@@ -940,6 +942,19 @@ export const generate = (baseSpacing = 24, scale = 6) => {
       // colors: [] || colors: ['graph-0', 'graph-1', 'graph-2', 'graph-3'],
       // extend: undefined,
     },
+    pagination: {
+      // container: {
+      //   // any box props,
+      //   // extend: undefined,
+      // },
+      icons: {
+        // color: undefined,
+        first: ChapterPrevious,
+        last: ChapterNext,
```

Comment:
Just noting that we will want to add more suitable icons.

```suggestion
        first: ChapterPrevious, // placeholder icon for discussion purposes
        last: ChapterNext, // placeholder icon for discussion purposes
```

Reply:
Done.